packages/Filters/src/Filters.js

```diff
@@ -0,0 +1,148 @@
+import React from "react";
+import PropTypes from "prop-types";
+import AddIcon from "@paprika/icon/lib/Add";
+import Button from "@paprika/button";
+import FilterIcon from "@paprika/icon/lib/Filter";
+import Panel from "@paprika/panel";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Item from "./components/Item";
+import FilterContext from "./context";
+import columnShape from "./columnShape";
+import * as types from "./types";
+
+import * as sc from "./Filters.styles";
+
+function getLabelText(numberOfFilters, I18n) {
+  switch (numberOfFilters) {
+    case 0:
+      return I18n.t("filters.label");
+    case 1:
+      return I18n.t("filters.singular_label");
+    default:
+      return I18n.t("filters.plural_label", { numberOfFilters });
```

Comment:
This used to be done automatically, but i haven't done it in at well over a year.  I think all you have to do is:
```yaml
// en.yml
filters:
  label: filter
  label_plural: {count} filters
```
```javascript
//Filters.js
I18n.t("filters.label", { count: numberOfFilters });
```
and it will choose the proper one automatically.
Have to do something specific for the "0" case, though.

https://www.i18next.com/translation-function/plurals

Reply:
I didn't know i18n can handle that! I'll try it out thank you!